src/pages/index.js

```diff
@@ -77,15 +134,28 @@ profileAddButton.addEventListener("click", () => {
 const profileEditPopup = new PopupWithForm(
   {
     popupSelector: "#profile-edit-modal",
-    handleFormSubmit: ({ title, description }) => {
-      profileInfo.setUserInfo(title, description);
-      profileEditPopup.close();
-    }
+    handleFormSubmit: ({ title, description }, profileEditForm) => {
+      profileEditPopup.renderLoading(true);
+      api.updateProfile(title, description)
+        .then((data) => {
+          profileInfo.setUserInfo(data.name, data.about, data.avatar),
+            profileEditForm.reset;
```

Comment:
`profileEditForm.reset;` doesn't work, and you don't need it

You shouldn't reset the profile form after submission - it has no effect because you fill the profile inputs with valid data on opening anyway


```js

      .updateProfile(title, description)
      .then((data) => {
        profileInfo.setUserInfo(data.name, data.about, data.avatar);
        profileEditPopup.close();
      })
      .catch((err) => {
        console.error("Update card error:", err);
      })
      .finally(() => {
```